app/src/main/java/org/gdg/frisbee/android/app/App.java

```diff
@@ -194,12 +196,29 @@ private void initTaggedEventSeries() {
             Const.START_TIME_GCP_NEXT,
             Const.END_TIME_GCP_NEXT));
 
+        updateEventSeriesAlarms();
+
+    }
+
+    private void updateEventSeriesAlarms() {
+        if (isTranslationForAlarmsAvailable()) {
+            for (TaggedEventSeries eventSeries : currentTaggedEventSeries()) {
+                NotificationHandler notificationHandler = new NotificationHandler(this, eventSeries);
+                if (notificationHandler.shouldSetAlarm()) {
+                    notificationHandler.setAlarmForNotification();
+                }
+            }
+        }
+    }
+
+    private boolean isTranslationForAlarmsAvailable() {
```

Comment:
Should we have this? I think English notification would be fine. Lots of users will miss the feature if we will have this.